### PR TITLE
Disable update checking in stripe listen command

### PR DESCRIPTION
**Pull request by ritikbhatia**
### Reviewers
r? @richardm-stripe @tomer-stripe 
cc @stripe/developer-products

 ### Summary
This pull request is a potential solution to PR #496 
This PR adds the `--skip-update` / `-s` flag to allow the user to skip checking for update. This will help improve the speed by avoiding the overhead of checking for new updates.

Example commands for usage of this flag:

- `stripe listen --skip-update`
- `stripe listen -s`

### CODE REVIEW

**Review by pepin-stripe — APPROVED** [review]
hey @ritikbhatia thanks for the PR! Looks great, feel free to merge and it will get into the next release!

**Comment by ritikbhatia** [comment]
Hi @pepin-stripe! Thanks for approving the PR! I cannot merge the PR since I do not have write access to the repository. Could you (or anyone in the Stripe team with write access to the repo) merge it?

**Pull request merged**
